### PR TITLE
Fix effect docs in microcosm api documentation

### PR DESCRIPTION
**Pull request by nhunzaker**
Thanks for catching this, @cwmanning

### CODE REVIEW

**Comment by leobauza** [comment]
Noting that this addresses parts of #250

File: docs/api/microcosm.md [hunk 1]
```diff
@@ -104,20 +104,38 @@ manage the provided `key`. Whenever this domain responds to an action,
 it will be provided the current state for that particular key.
 
 `options` passed as the third argument are sent into a domain's
-`setup` method and, if using a class, the constructor.
+`setup` method and, if using a class, the constructor is instantiated
+with the provided options and associated repo.
```
Comment (leobauza) [inline docs/api/microcosm.md]: Why is there no code sample for `addDomain` like there is for `addEffect` ?

Comment (nhunzaker) [inline docs/api/microcosm.md]: There is no reason. I'll add an issue for this. We also need to better describe how you can create domains and effects (simple objects vs classes)

**Comment by nhunzaker** [comment]
@leobauza I merged this, I'll follow up with another PR for the domains stuff.

**Pull request closed without merging**